packages/suite/src/hooks/suite/useLocales.ts

```diff
@@ -12,13 +12,15 @@ export const useLocales = () => {
     useEffect(() => {
         let active = true;
         const loadLocale = async () => {
-            const lang = language === 'en' ? 'en-US' : language;
+            const lang = language === 'en' ? 'enUS' : language;
 
             let dateLocale;
             try {
-                dateLocale = await import(/* @vite-ignore */ `date-fns/locale/${lang}/index`);
+                dateLocale = await import(/* @vite-ignore */ `date-fns/locale`).then(
+                    module => module[lang as keyof typeof module],
```

Comment:
And it's weird that you can't just import just the language you want and not any other.